test-e2e/members.js

```diff
@@ -0,0 +1,144 @@
+import { test } from 'brittle'
+import RAM from 'random-access-memory'
+import { KeyManager } from '@mapeo/crypto'
+import pDefer from 'p-defer'
+
+import { MapeoManager, kRPC } from '../src/mapeo-manager.js'
+import {
+  CREATOR_CAPABILITIES,
+  DEFAULT_CAPABILITIES,
+  MEMBER_ROLE_ID,
+} from '../src/capabilities.js'
+import { replicate } from '../tests/helpers/rpc.js'
+
+test('getting yourself', async (t) => {
+  const { manager } = setup()
+
+  await manager.setDeviceInfo({ name: 'mapeo' })
+  const project = await manager.getProject(await manager.createProject())
+  await project.ready()
+
+  const me = await project.$member.getById(project.deviceId)
+
+  t.alike(me, {
+    deviceId: project.deviceId,
+    name: 'mapeo',
+    capabilities: CREATOR_CAPABILITIES,
+  })
+
+  const members = await project.$member.getMany()
+
+  t.is(members.length, 1)
+
+  t.alike(members[0], me)
+})
+
+// TODO: Fails. Needs MemberApi.invite() to be updated to write device info record of invited peer
+test('getting others after invite', { todo: true }, async (t) => {
```

Comment:
I think this test fails to run due to some additional work needed in the MemberApi implementation, namely saving the saving the invited member's device info when they accept.

Reply:
- DeviceInfo and CoreOwnership records need to be written by the invited peer
- The invitor needs to write the Role record
- They need to sync after invite so each gets these records
-